tests/packagedcode/data/docker/test.containerfile

```diff
@@ -0,0 +1,80 @@
+#Copied from https://github.com/kubernetes-sigs/blixt/blob
+
+
+FROM rust:1.79-slim-bookworm as builder
+
+ARG TARGETARCH
+ARG LLVM_VERSION=19
+
+RUN apt-get update
+RUN apt-get install --yes \
+    build-essential \
+    protobuf-compiler \
+    pkg-config \
+    musl-tools \
+    clang \
+    wget
+
+RUN apt install --yes lsb-release software-properties-common gnupg
+RUN wget -O /tmp/llvm.sh https://apt.llvm.org/llvm.sh
+RUN chmod +x /tmp/llvm.sh
+RUN /bin/sh -c "/tmp/llvm.sh ${LLVM_VERSION} all"
+
+RUN rustup default stable
+RUN rustup install nightly
+RUN rustup component add rust-src --toolchain nightly
+RUN --mount=type=cache,target=/root/.cargo/registry \
+    cargo install bpf-linker
+
+WORKDIR /workspace
+# Docker uses the amd64/arm64 convention while Rust uses the x86_64/aarch64 convention.
+# Since Dockerfile doesn't support conditional variables (sigh), write the arch in Rust's
+# convention to a file for later usage.
+RUN if [ "$TARGETARCH" = "amd64" ]; \
+    then echo "x86_64" >> arch; \
+    else echo "aarch64" >> arch; \
+    fi
+RUN rustup target add $(eval cat arch)-unknown-linux-musl
+
+COPY dataplane dataplane 
+COPY tools/udp-test-server tools/udp-test-server
+COPY xtask xtask
+COPY Cargo.toml Cargo.toml
+COPY Cargo.lock Cargo.lock
+COPY .cargo .cargo
+
+# We need to tell bpf-linker where it can find LLVM's shared library file.
+# Ref: https://github.com/aya-rs/rustc-llvm-proxy/blob/cbcb3c6/src/lib.rs#L48
+ENV LD_LIBRARY_PATH="/usr/lib/llvm-$LLVM_VERSION/lib"

```

Comment:
it might make sense to trim a little bit the parts of this file which are not useful for it's tests to make it smaller (in a way that doesn't give errors on the parsing). The repo quickly gets large because of large test files, as we have several thousands of these at these point, and it adds up to the repo size.